addons/l10n_latam_check/tests/test_third_party_checks.py

```diff
@@ -88,60 +88,35 @@ def test_02_third_party_check_delivery(self):
         with self.assertRaisesRegex(ValidationError, "Some checks are not anymore in journal,"), self.cr.savepoint():
             self.env['account.payment'].create(vals).action_post()
 
-        operations = self.env['account.payment'].search([('l10n_latam_move_check_ids', '=', check.id), ('state', '=', 'posted')], order="date desc, id desc")
+        operations = self.env['account.payment'].search([('l10n_latam_move_check_ids', '=', check.id), ('state', '!=', 'draft')], order="date desc, id desc")
         self.assertEqual(len(operations), 3, 'There should be 3 operations on the check')
         self.assertEqual(operations, customer_return | supplier_return | delivery)
 
     def test_03_check_deposit(self):
         payment = self.create_third_party_check()
         check = payment.l10n_latam_new_check_ids[0]
         bank_journal = self.company_data_3['default_journal_bank']
-
         # Check Deposit
         deposit = self.env['l10n_latam.payment.mass.transfer'].with_context(
-            active_model='l10n_latam.check', active_ids=[check.id]).create({'destination_journal_id': bank_journal.id})._create_payments()
+            active_model='l10n_latam.check',
+            active_ids=[check.id]
+        ).create({
+            'destination_journal_id': bank_journal.id,
+        })._create_payments()
         self.assertEqual(check.current_journal_id, bank_journal, 'Current journal was not computed properly on delivery')
-
         # check dont deposit twice
         with self.assertRaisesRegex(UserError, "All selected checks must be on the same journal and on hand"), self.cr.savepoint():
             self.env['l10n_latam.payment.mass.transfer'].with_context(
-                active_model='l10n_latam.check', active_ids=[check.id]).create({'destination_journal_id': bank_journal.id})._create_payments()
-
+                active_model='l10n_latam.check',
+                active_ids=[check.id]
+            ).create({
+                'destination_journal_id': bank_journal.id,
+            })._create_payments()
         # Check Rejection
-        vals = {
-            'l10n_latam_move_check_ids': [Command.set([check.id])],
-            'payment_type': 'inbound',
-            'journal_id': self.rejected_check_journal.id,
-            'payment_method_line_id': self.rejected_check_journal._get_available_payment_method_lines('inbound').filtered(lambda x: x.code == 'in_third_party_checks').id,
-        }
-        bank_rejection = self.env['account.payment'].create(vals)
-        bank_rejection.action_post()
-        self.assertEqual(check.current_journal_id, self.rejected_check_journal, 'Current journal was not computed properly on return')
-        # check dont reject twice
-        with self.assertRaisesRegex(ValidationError, "it seems it has been moved by another payment"), self.cr.savepoint():
-            self.env['account.payment'].create(vals).action_post()
+        #TODO
```

Comment:
We decided with CHKL that we will remove this test since now it will go through the wizard every time so i will remove it once it's good functionnaly by valentino